drools-wb-screens/drools-wb-guided-template-editor/drools-wb-guided-template-editor-client/src/main/java/org/drools/workbench/screens/guided/template/client/editor/TemplateDataCellFactory.java

```diff
@@ -63,6 +64,10 @@ public TemplateDataCellFactory(AsyncPackageDataModelOracle oracle,
      */
     public DecoratedGridCellValueAdaptor<? extends Comparable<?>> getCell(TemplateDataColumn column) {
 
+        if(column.getDataType().equals(TemplateModel.DEFAULT_TYPE)){
```

Comment:
Unfortunately you need to do something similar for the Guided Decision Table's use of "BRL Fragments".

I had a quick look around for _where_ a similar change is needed, but unfortunately my brain let me down.

I'd put a break point in `BRLConditionVariableColumnConverter` and see where it leads.

Reply:
No worries, good thing you spotted this. I'll look for it.

Reply:
BRLConditionColumnPlugin.getDefinedVariables(RuleModel ruleModel) is where I need to add some changes. Adding this here as a reminder for myself ;)